docs/compose-file.md

```diff
@@ -577,6 +603,211 @@ documentation for more information. Optional.
         baz: 1
 
 
+## Network configuration reference
+
+The top-level `networks` key lets you specify networks to be created. For a full
+explanation of Compose's use of Docker networking features, see the
+[Networking guide](networking.md).
+
+### driver
+
+Specify which driver should be used for this network. Defaults to `bridge`. The
+Docker Engine will return an error if the driver is not available.
```

Comment:
I believe it actually defaults to `overlay` on swarm, so we might want to say the default depends on the engine.